src/vpn-api/api.ts

```diff
@@ -123,10 +123,11 @@ interface ExpressAsync extends express.Express {
 export const apiServer = (instanceId: number) => {
 	const app = Promise.promisifyAll(express()) as ExpressAsync;
 	app.disable('x-powered-by');
+	app.use(Sentry.Handlers.requestHandler());
```

Comment:
We removed this when using raven because it enables/uses domains which causes a memory leak - did that get changed so it no longer uses domains?

Reply:
It still uses `domain` but the implementation pretty much unrecognisable from `raven`. Is the use of `domain` going to be a problem full-stop?

Reply:
Interesting that they opted to use `domain` in the newer module despite it being deprecated. I suppose it makes sense if they're doing some fancy error-juggling and there isn't a replacement api available in node yet.

Reply:
Yeah, there isn't really an alternative to domains for what they're trying to do, but the memory leak is in domains themselves so any usage will be problematic for us

Reply:
Gotcha, I'll nix that handler for now then!